edx_repo_tools/data.py

```diff
@@ -28,9 +29,30 @@ def iter_nonforks(hub, orgs):
                 yield repo
 
 
-def iter_openedx_yaml(hub, orgs, branches=None):
+def find_file_content(repo, branch):
     """
-    Yield the data from all openedx.yaml files found in repositories in ``orgs``
+    Finds the data from all catalog-info.yaml or openedx.yaml files found in repositories in ``repo``
+    on any of ``branches``.
+    First, we check for catalog-info.yaml, If catalog-info.yaml is not found, we look for openedx.yaml file.
+
+    """
```

Comment:
The function signature and code suggest that it is a single repo and branch but the docstring seems to be referring to multiple repos and branches, please update the docstring.

Reply:
I removed this method when resolving the review comments and code logic.